internal/servers/server/option_test.go

```diff
@@ -0,0 +1,1673 @@
+package server
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/vdaas/vald/internal/errgroup"
+	"github.com/vdaas/vald/internal/errors"
+	"google.golang.org/grpc"
+)
+
+func TestWithHost(t *testing.T) {
+	type test struct {
+		name      string
+		host      string
+		checkFunc func(opt Option) error
+	}
+
+	tests := []test{
+		{
+			name: "set success",
+			host: "host",
+			checkFunc: func(opt Option) error {
+				got := new(server)
+				opt(got)
+
+				if got.host != "host" {
+					return errors.New("invalid param was set")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "do nothing",
```

Comment:
- I think there may be a better name for this test, ex) `return(s) nothing`. Could you tell me your opinion ?

Reply:
thank you for your review!
i think so. I will fix soon